go-controller/hack/gosec.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+GO111MODULE=on ${GOPATH}/bin/gosec ./... 
```

Comment:
Might be worth adding a comment on where to get gosec for anyone wanting to run it outside of CI. 

The quickest install I saw was:

```
curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $GOPATH/bin latest
```